readme.md

```diff
@@ -45,6 +29,22 @@ For a more in depth explanation as to why the service is doing this, please see
   - [Schedule<sup>1</sup>](#schedulesup1sup)
     - [Planned Release Schedule](#planned-release-schedule)
 
+## Critical Upcoming Change Notice
+
+All Azure IoT SDK users are advised to be aware of upcoming TLS certificate changes for Azure IoT Hub and Device Provisioning Service 
+that will impact the SDK's ability to connect to these services. In October 2022, both services will migrate from the current 
+[Baltimore CyberTrust CA Root](https://baltimore-cybertrust-root.chain-demos.digicert.com/info/index.html) to the 
+[DigiCert Global G2 CA root](https://global-root-g2.chain-demos.digicert.com/info/index.html). There will be a 
+transition period beforehand where your IoT devices must have both the Baltimore and Digicert public certificates 
+hardcoded in their application or flashed onto your WiFi module in order to prevent connectivity issues. 
```

Comment:
```suggestion
which maybe hardcoded in their application or flashed onto your WiFi module in order to prevent connectivity issues. 
```

Reply:
"which may be...", but yeah I like this change